controllers/oapserver_controller.go

```diff
@@ -52,6 +52,7 @@ type OAPServerReconciler struct {
 // +kubebuilder:rbac:groups=operator.skywalking.apache.org,resources=oapservers,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=operator.skywalking.apache.org,resources=oapservers/status,verbs=get;update;patch
 // +kubebuilder:rbac:groups=apps,resources=deployments,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups=,resources=services,verbs=get;list;watch;create;update;patch;delete
```

Comment:
```suggestion
// +kubebuilder:rbac:groups="",resources=services,verbs=get;list;watch;create;update;patch;delete
```

 Need to specify group.
```
The ClusterRole "skywalking-swck-manager-role" is invalid: rules[0].apiGroups: Required value: resource rules must supply at least one api group
```